src/lib/molecules/Charts/MapChartD3/MapChartD3.stories.tsx

```diff
@@ -0,0 +1,120 @@
+import React, { FC } from 'react';
+import { Meta } from '@storybook/react';
+import { args, propCategory } from '../../../../../stories/assets/storybook.globals';
+
+// Components
+import MapChartD3, { IMapChartD3Props } from './index';
+
+// Data
+import { regionData as testData, worldMapData } from '__data__';
+
+const regionData = [
+    {
+        id: 'AM.ER',
+        value: 12
+    },
+    {
+        id: 'AM.TV',
+        value: 54
+    },
+    {
+        id: 'AM.SU',
+        value: 73
+    },
+    {
+        id: 'AM.SH',
+        value: 17
+    },
+    {
+        id: 'AM.GR',
+        value: 35
+    }
+];
+
+const meta: Meta<typeof MapChartD3> = {
+    title: 'Charts/MapChartD3',
+    component: MapChartD3,
+    argTypes: {
+        title: args({ control: 'text', ...propCategory.content }),
+        viewActivityContent: args({ control: false, ...propCategory.content }),
+        viewActivityName: args({ control: 'text', ...propCategory.content }),
+        withNavigation: args({ control: 'boolean', ...propCategory.functionality }),
+        withActivity: args({ control: 'boolean', ...propCategory.states }),
+        regionData: args({ control: 'object', ...propCategory.content }),
+        tooltipRenderer: args({ control: false, ...propCategory.content }),
+        withLegend: args({ control: 'boolean', ...propCategory.functionality }),
+        brightness: args({ control: 'number', ...propCategory.appearance }),
```

Comment:
This prop in storybook changes nothing

Reply:
Because brightness should work only on component mount.